profiles/package.mask

```diff
@@ -1,3 +1,11 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-09)
+# Per the description, this is simply a "convenient wrapper" around some other
+# diagrams-* packages. This is currently masked due to being unmaintained, and
+# not pulling in newer versions of the packages it wraps. An upstream issue has
```

Comment:
Hm, i considered this, but thought since the package was out-of-date (and the author agreed it was under maintained) that it wouldn't be worth the effort.

I'll give it a whirl though it should be rather straightforward.

Reply:
Done,  2fd7692ef6051eb2a4c1ff30f4c08881e25319f4